### ISSUE TITLE
ClojureScript self-host compatible libs

### ISSUE THREAD

**Issue by arichiardi** [weavejester/clojure-toolbox.com#256]
Hello James!

So for me one of the biggest oddities of ClojureScript for novice folks is the difference between JVM vs self-host.

The problem is in the dependencies, where it is not always true that libraries for ClojureScript JVM are compatible with self-host. Actually this is rarely the case.

So my issue is about the platform. We could introduce a `cljs-self-host` (or shorter name) platform in order to flag the libs that are compatible with it.

The solution seems easy but maybe I am overlooking something in your data model.

**Comment by weavejester** [comment]
I'm not against this in principle, but using a tag to communicate this information has two problems that I can see. Both "clj" and "cljs" are concise and self-explanatory. A tag like "cljs-self-host" is neither of these things. Perhaps we could introduce a key at the top explaining the tags somehow, but that would require more UI work.

**Comment by arichiardi** [comment]
Yeah well the UI can be can be an overlay icon. Not happy about this name and approach either by the way, that is was I was seeking some advice.
